docker/deployer/Dockerfile

```diff
@@ -1,5 +1,14 @@
+# trunk-ignore-all(terrascan/AC_DOCKER_0047)
+# trunk-ignore-all(hadolint/DL4006)
+# trunk-ignore-all(hadolint/DL3059)
 FROM python:3.9.13
 
+HEALTHCHECK NONE
+
+RUN groupadd -r appgroup && useradd -r -g appgroup appuser
```

Comment:
what is this for?

Reply:
it's to remove a warning from trunk :/ 

HEALTHCHECK will monitor the docker health (not needed here because the deployer just needs to run once).